<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfacesVlanEdit</class>
 <widget class="QMainWindow" name="InterfacesVlanEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="allZonesList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dragOneRightBrn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dragOneLeftBrn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="mode">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Access</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Trunk</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="selectedZonesList">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commitBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Commit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dragAllRightBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dragAllLeftBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
